index.js

```diff
@@ -498,10 +498,18 @@ prototype.buildOperation = function(oldOperation, operationDefaults) {
     parameters.push(this.buildParameter(oldParameter));
   });
 
+  var tags = (operationDefaults.tags || []).concat(oldOperation.tags || []);
+
+  tags = tags.filter(function(e, i, arr) {
```

Comment:
@ceritium 
I think finding duplicates is generic enough to move it into separate function.
This code isn't simple to understand from first glance, but `tags = removeDuplicates(tags)` is much easier.